aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -93,5 +93,8 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.aggregator.read.base, "aggregator.read.base", 1],
     [.aggregator.sub.base, "aggregator.sub.base", 1],
     [.aggregator.destroy.base, "aggregator.destroy.base", 1],
-    [.aggregator_factory.new_aggregator.base, "aggregator_factory.new_aggregator.base", 1]
+    [.aggregator_factory.new_aggregator.base, "aggregator_factory.new_aggregator.base", 1],
+
+    [.network_address.validate_network_addresses.base, "network_address.validate_network_addresses.base", 1],
+    [.network_address.validate_network_addresses.per_byte, "network_address.validate_network_addresses.per_byte", 1],
```

Comment:
Just a reminder that when introducing new gas parameters, we need to add them to the on-chain gas schedule first before we could update the rust binary, or otherwise the VM will fail to start.